rules/onpremises-kfd-v1alpha2.yaml

```diff
@@ -25,6 +25,12 @@ distribution:
     reducers:
       - key: distributionModulesLoggingType
         lifecycle: pre-apply
+  - path: .spec.distribution.modules.monitoring.alertmanager.defaultRules
+    immutable: false
+    description: "changes to the monitoring alertmanager configs have been detected. This will cause a deletion of the defaults alertmanager configs."
```

Comment:
```suggestion
    description: "changes to the monitoring alertmanager configs have been detected. This will cause the reconfiguration (creation or deletion) of the defaults alertmanager configs."
```